charts/argo-cd/values.yaml

```diff
@@ -3755,6 +3755,7 @@ notifications:
 
 eventReporter:
   name: event-reporter
+  enabled: true
```

Comment:
probably you wan't disable it by default no ?

Reply:
not right now. i want our default argo-cd fork to behave like it always does, and then i'll be able to play around with that value without interfering with anything else